xbmc/cores/VideoPlayer/DVDFileInfo.cpp

```diff
@@ -454,11 +455,11 @@ bool CDVDFileInfo::DemuxerToStreamDetails(CDVDInputStream *pInputStream, CDVDDem
   // correct bluray runtime. we need the duration from the input stream, not the demuxer.
   if (pInputStream->IsStreamType(DVDSTREAM_TYPE_BLURAY))
   {
-    if(((CDVDInputStreamBluray*)pInputStream)->GetTotalTime() > 0)
+    if (static_cast<CDVDInputStreamBluray*>(pInputStream)->GetTotalTime() > 0)
     {
-      CStreamDetailVideo* detailVideo = (CStreamDetailVideo*)details.GetNthStream(CStreamDetail::VIDEO, 0);
+      CStreamDetailVideo* detailVideo = static_cast<CStreamDetailVideo*>(details.GetNthStream(CStreamDetail::VIDEO, 0));
```

Comment:
this produces an error (cast const CStreamDetailVideo to non const CStreamDetailVideo)